### PR TITLE
Add news file

### PR DESCRIPTION
**Pull request by bam241**
adding a news file to `pymoab_cleanup`

I never know how to fill those files properly.... so please let me know what I did not do right...

### CODE REVIEW

File: news/pymoab_cleanup.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+   - mesh.destroyTag(self, boolean) -> mesh.delete_tag(self)
+   - mesh.tag.type -> mesh.tag.get_dtype()
+   - write_hdf5(self, filename) -> write_hdf5(self, filename, write_mats)
+   - imesh.iterate() -> mesh.mesh_iterate()
+   - iMesh.Mesh().load(hdf5) -> core.Core().load_file(hdf5)
+   - iMesh.Mesh().getEntSets() -> core.Core().get_entities_by_type(0, types.MBENTITYSET)
+   - iMesh.Mesh().getTagHandle('CATEGORY') -> core.Core().get_entities_by_type(0, types.MBENTITYSET)
+   - iMesh.Mesh().getTagHandle('GLOBAL_ID') -> core.Core().tag_get_handle(types.GLOBAL_ID_TAG_NAME)
+   - iMesh.Mesh().getTagHandle('NAME') -> core.Core().tag_get_handle(types.NAME_TAG_NAME)
+   - iMesh.mesh.getTagHandle('cell_number') -> mesh.cell_number
```
Comment (gonuke) [inline news/pymoab_cleanup.rst]: These changes are not due to pyMOAB. This was already available in IMeshTag but was rarely used.

When the tag is created on a mesh object, it is added to the attribute list of that mesh object.

File: news/pymoab_cleanup.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+   - mesh.mesh.getTagHandle('idx') -> mesh.idx
+   - idx.append(mesh.mesh.getTagHandle('idx')[ve]) -> idx.append(mesh.idx[ve][0])
+   - del mtag[XXXX] -> mesh.mesh.tag_delete_data(mtag, XXXX)
```
Comment (gonuke) [inline news/pymoab_cleanup.rst]: We should make sure that this is correct with respect to #29

File: news/pymoab_cleanup.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+   - mesh.getAllTags(mesh.rootSet) -> mesh.get_all_tags()
+   - mesh.getAllTags('idx') -> mesh.tag_get_handle('idx')
+   - mesh.getAllTags(ve) -> mesh.tag_get_tags_on_entity(ve)
+   - mesh.createTag(tag_name, num_e_groups * max_num_cells, float) -> mesh.tag(tag_name, np.zeros(float_tag_size, dtype = float), 'nat_mesh', size = float_tag_size, dtype = float)
```
Comment (gonuke) [inline news/pymoab_cleanup.rst]:
```suggestion
   - mesh.createTag(tag_name, length_of_vector, float) -> mesh.tag(tag_name, np.zeros(length_of_vector, dtype = float), 'nat_mesh', size = length_of_vector, dtype = float)
```

**Review by gonuke — CHANGES_REQUESTED** [review]
We probably want a little more commentary on each of these or on subsets of these to describe which use case this is for...

File: news/pymoab_cleanup.rst [hunk 4]
```diff
@@ -0,0 +1,54 @@
+**Added:** None
+
+* Testing PyNE build without PyMOAB using Python 2 & 3
+* Added FindDAGMC.cmake file
+
+**Changed:** 
+
+* PyNE can be build with PyMOAB whitout DAGMC with limited capabilities
```
Comment (ngranda) [inline news/pymoab_cleanup.rst]:
```suggestion
* PyNE can be built with PyMOAB whitout DAGMC with limited capabilities
```

**Comment by bam241** [comment]
@gonuke do you like that better ?
(this is not necessary finished yet...)

**Review by gonuke — APPROVED** [review]

**Comment by gonuke** [comment]
thanks @bam241

**Pull request merged**